nemo/collections/asr/parts/utils/audio_utils.py

```diff
@@ -378,3 +379,56 @@ def pow2db(power: float, eps: Optional[float] = 1e-16) -> float:
         Power in dB.
     """
     return 10 * np.log10(power + eps)
+
+
+def get_segment_start(signal: np.ndarray, segment: np.ndarray) -> int:
+    """Get starting point of `segment` in `signal`.
+    We assume that `segment` is a subset of `signal`.
+
+    Args:
+        signal: numpy array with shape (num_samples,)
+        segment: numpy array with shape (num_samples,)
+
+    Returns:
+        Index of the start of `segment` in `signal`.
+    """
+    assert len(signal) >= len(segment), 'segment must be shorter than signal'
+    cc = scipy.signal.correlate(signal, segment, mode='valid')
+    return np.argmax(cc)
+
+
+def calculate_sdr(
```

Comment:
Is this different from SDR calculation in audio_losses.py?

Reply:
if this is used only for testing purpose, can we move it to `tests`?

Reply:
Lets keep it here but make the name clear that it is sample level inefficient numpy code